xbmc/guilib/GUIFontManager.cpp

```diff
@@ -492,8 +492,7 @@ void GUIFontManager::SettingOptionsFontsFiller(const SettingConstPtr& setting,
     {
       CFileItemPtr pItem = items[i];
 
-      if (!pItem->m_bIsFolder
-          && URIUtils::HasExtension(pItem->GetLabel(), ".ttf"))
+      if (!pItem->m_bIsFolder && URIUtils::HasExtension(pItem->GetLabel(), ".ttf|.otf"))
```

Comment:
Here we should also try to avoid redefining the same logic. Maybe create a new method in Utils that is able to receive the path. Then rewrite the IsUsingTTFsubtitles() method to use that new method by providing the setting path.
Actually thinking a bit more about this, the IsUsingTTFSubtitles() (or equivalent) belong to the new subtitlessettings class you introduced with the ass rework. The main logic of that method (the actual uriutils logic) belongs to utils.
Not sure I expressed myself in a clear way but please feel free to reach out if not

Reply:
yes i think to understand i will prepare the things

Reply:
i have not followed in full your tips, because Fonts "filler" can be used also for other purposes different from subtitles, then i have changed the method in to more generic `IsSupportedFontExtension`,
Instead in the SettingConditions (the previous IsUsingTTFSubtitles) the behaviour is not more hardcoded to the subtitle setting but will depends on the `setting` condition specified in the xml, in this case subtitles.font:
`<condition on="property" name="HasSubtitlesFontExtensions" setting="subtitles.font"/>`